package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "description": "Small description for gsoc-prep-tasks goes here",
-  "license": "MIT",
+  "license": "GPL-3.0-only",
```

Comment:
LICENSE.md said `either version 3 of the License, or (at your option) any later version.`

Should be `or-later`, see header of https://spdx.org/licenses/GPL-3.0-or-later.html#licenseText

Although, GPL-3.0-only also said `either version` https://spdx.org/licenses/GPL-3.0-only.html#licenseText ,
the header is different.

Reply:
updated

Reply:
We can ignore that bit, but it will be relevant to any attempt at automating the LICENSE files via moban :/

The important part is that the GPL-3.0-only *requires* the preamble which says ', version 3.' and this repo LICENSE doesnt have that.

See placement of `<standardLicenseHeader>` in
https://github.com/spdx/license-list-XML/blob/master/src/GPL-3.0-only.xml
whereas `<standardLicenseHeader>` is in `<optional>` in
https://github.com/spdx/license-list-XML/blob/master/src/GPL-3.0-or-later.xml